doc/overview.rst

```diff
@@ -1,98 +1,106 @@
 
-  * MATLAB-like PYTHONPATH management dialog box (works with all consoles)
-  * Windows only: current user environment variables editor
-  * direct links to documentation (Python, Matplotlib, !NumPy, !Scipy, etc.)
-  * direct link to Python(x,y) launcher
-  * direct links to !QtDesigner, !QtLinguist and !QtAssistant (Qt documentation)
+  * Occurrence highlighting: Select or double-click any word to show all other instances throughout the current document
+  * On-the-fly automatic formatters (optional):
 
-* *preferences* dialog box:
+    * Automatic insertion of closing quotes parentheses, braces and brackets
+    * Automatic indentation after 'else', 'elif', 'finally', etc.
+    * Smart auto-indentation based on code structure
+    * Automatic insertion of colons after for, if, def, etc.
+    * Automatically fix mixed indentation, EoL characters and trailing spaces
 
-  * keyboard shortcuts
-  * syntax coloring schemes (source editor, history log, help)
-  * console: background color (black/white), automatic code completion, etc.
-  * and a lot more...
+  * Real-time code analysis:
+    * Errors/warnings/problems (powered by ``pyflakes``)
+    * PEP 8 and code style (powered by ``pycodestyle``)
+    * Code annotation parsing and "Todo lists" (``TODO``, ``FIXME``, ``XXX``, etc.)
 
 .. image:: images/mainwindow/mainwindow_dark_1610.png
    :align: right
    :width: 60%
    :alt: The Spyder main window, with a dark syntax highlighting theme applied
 
+* :doc:`ipythonconsole`:
 
-* :doc:`editor`:
-
-  * syntax coloring (Python, C/C++, Fortran)
-  * *breakpoints* and *conditional breakpoints* (debugger: `pdb`)
-  * run or debug Python scripts (see console features)
-  * *run configuration* dialog box:
-
-    * working directory
-    * command line options
-    * run in a new Python interpreter or in an existing Python interpreter or IPython client
-    * Python interpreter command line options
+  * Any number of individual consoles, each executed in a separate, isolated processes
+  * Each console uses the full IPython kernel as a back-end with a light GUI front-end
+  * Supports all of the powerful IPython magic commands and functionality
+  * Automatic code completion and calltips, and automatic link to :doc:`help`
+  * Range of code run options and interactivity
+  * :doc:`debugging` integration with enhanced ``ipdb`` debugger and the :doc:`editor`
+  * Inline display of Matplotlib graphics (optional)
+  * The :guilabel:`User Module Reloader`, automatically re-importing modified source files
 
-  * *code outline explorer*: functions, classes, if/else/try/... statements
-  * *powerful code introspection features* (powered by `rope`):
+* :doc:`variableexplorer`:
 
-    * *code completion*
-    * *calltips*
-    * *go-to-definition*: go to object (any symbol: function, class, attribute, etc.) definition by pressing Ctrl+Left mouse click on word or Ctrl+G (default shortcut)
+  * Lists all global variables, functions, classes, and their content
```

Comment:
This is not true by default. What we show by default right now is iterables (lists, dicts, tuples), Numpy arrays and Dataframes.

Reply:
Perhaps it could be enough to remove `all` here, to not give the false impression that we show everything by default.